tests/remote/test_fennec.py

```diff
@@ -17,7 +17,8 @@
     # -p android-api-9 --branch=mozilla-central
     {'args': {'application': 'fennec',
               'platform': 'android-api-9',
-              'branch': 'mozilla-central'}},
+              'branch': 'mozilla-central',
+              'date': '2016-01-01'}},
```

Comment:
For reference and in case we would need that day in the future eg. to determine which service to use for scraping for the build.